README.md

```diff
@@ -27,3 +27,9 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+The invariant for the Fibonacci sequence is that the first two elements of the array must be 0 and 1. This must always be true at the start of the recursive call to get out whatever size of Fibonacci sequence you desire. This is because the Fibonacci sequence is based on adding the last two elements in the sequence to get the next number in the sequence.  And so, to get any element inside the Fibonacci sequence, you must have the two preceding elements, meaning that the absolute minimum amount of information you need to produce any element inside the Fibonacci sequence is the first two elements, 0 and 1. Then, you can build your way up to any size of the Fibonacci sequence you desire.
```

Comment:
And what has to be true at the beginning of each recursive call before you compute the current Fibonacci number?